CRFModuleValidation/ScheduledActivityManager.swift

```diff
@@ -119,6 +119,40 @@ class ScheduledActivityManager: SBABaseScheduledActivityManager, SBAScheduledAct
         return nil
     }
     
+    func createRSDTaskViewController(for schedule: SBBScheduledActivity) -> RSDTaskViewController? {
```

Comment:
Does it need to be an RSDTaskViewController to be presented? Could it just vend a UIViewController? Then would work for both.

Reply:
This was me refactoring your didSelect... code so I could get an RSDTaskViewController for the MasterScheduledActivityManager, which doesn't need the other kind. But it probably does make sense to factor all the didSelect... logic out into that method and have it return a UIViewController, since the output of that method only ever does get presented so it doesn't need to be specifically one or the other.